inspectit-ocelot-agent/src/system-test/java/rocks/inspectit/ocelot/instrumentation/special/ExecutorContextPropagationTest.java

```diff
@@ -62,7 +62,7 @@ public void correlateRunnable_lambda() throws Exception {
 
             taskFuture.get();
 
-            assertThat(refTags.get()).hasSize(1)
+            assertThat(refTags.get()).toIterable().hasSize(1)
```

Comment:
That change was necessary to be able to use a current version of `assertj-core`.